hack/release.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -e
+
+released_version=$1
+future_version=$2
+origin_remote=$3
+fork_remote=$4
+
+echo 'Fetching origin'
+git fetch ${origin_remote}
+
+echo 'Checkout origin/master into release branch'
+git checkout ${origin_remote}/master -b release_${released_version}
+
+echo 'Build manifests for the new release'
+VERSION=${released_version} make generate-manifests
+
+echo 'Add new manifests to the source tree'
+git add manifests/
+
+echo 'Commit new manifests, this commit is to be tagged with the new release'
+git commit -s -m "release ${released_version} - update manifests"
+
+echo 'Bump versions in Makefile'
+sed -i "s/VERSION ?= .*/VERSION ?= ${future_version}/" Makefile
+sed -i "s/VERSION_REPLACES ?= .*/VERSION_REPLACES ?= ${released_version}/" Makefile
+
+echo 'Commit Makefile with bumped versions'
+git add Makefile
+
+echo 'Commit updated Makefile'
+git commit -s -m "release ${released_version} - bump versions in Makefile"
+
+echo 'Push changes to forked repo'
+git push ${fork_remote}
+
+echo 'The rest is on you, open a new PR. Once the PR is merged, do not forget to tag the first of these commits, include change log and upload released manifests'
```

Comment:
maybe change this to tag the commit started with `Merge pull request <pr-number>` ?

Reply:
I know it is a little weird, but release should be on the first one, where we generate manifests for the new version. The second commit is starting next version.

Practically there shouldn't be a difference. If you think it makes more sense to keep it on top of the PR, I will change it.